tests/www/companies_views/test_card_views.py

```diff
@@ -280,7 +280,15 @@ def test_card_tally_url_with_user(self):
         self.client.force_login(JobSeekerFactory(pk=10))
         response = self.client.get(url)
         soup = parse_response_to_soup(response, selector=".c-box--action")
-        assert str(soup) == self.snapshot()
+        assert str(soup) == self.snapshot(name="without_other_jobs")
+        # Create other job_description
+        JobDescriptionFactory(company=job_description.company)
+        response = self.client.get(url)
+        soup = parse_response_to_soup(response, selector=".c-box--action")
+        assert str(soup) == self.snapshot(name="with_other_jobs")
+        # Check link consistency
+        self.assertContains(response, 'href="#recrutements"')
+        self.assertContains(response, 'id="recrutements"')
```

Comment:
Le `href="#recrutements"` me semble redondant avec le snapshot donc peut-être juste tester la présence d'un élément avec l'id ?
```suggestion
        assert parse_response_to_soup(response, selector="#recrutements")
```